perf/benchmark/templates/fortio.yaml

```diff
@@ -149,26 +433,51 @@ spec:
         volumeMounts:
         - name: shared-data
           mountPath: /var/lib/fortio
+        - name: nighthawk-test-server-config
+          mountPath: /var/lib/nighthawk 
         image: {{ $.Values.curlImage }}
         args:
         - /bin/sleep
         - infinity
       - name: uncaptured
         securityContext:
           runAsUser: 1
+{{- if eq $.name "fortioclient" }}
         image: {{ $.Values.fortioImage }}
```

Comment:
@oschaaf I'm thinking here we should use nighthawk image not fortio, right?

Reply:
here might be a typo

Reply:
the idea with the  Fortio image here was that we could continue to leverage It for visualizing the plots via it’s reporting server

Reply:
oh, i see.